docs/api/system-preferences.md

```diff
@@ -326,3 +326,30 @@ You can use the `setAppLevelAppearance` API to set this value.
 
 Sets the appearance setting for your application, this should override the
 system default and override the value of `getEffectiveAppearance`.
+
+### `systemPreferences.hasMicrophoneAccess()` _macOS_
+
+Returns `Boolean` - `true` if the app has obtained system privacy consent to access the microphone and `false` otherwise.
+
+This user consent was not required until macOS 10.14 Mojave, so this method will always return `true` if your system is running 10.13 High Sierra or lower.
+
+### `systemPreferences.hasCameraAccess()` _macOS_
+
+Returns `Boolean` - `true` if the app has obtained system privacy consent to access the camera and `false` otherwise.
+
+This user consent was not required until macOS 10.14 Mojave, so this method will always return `true` if your system is running 10.13 High Sierra or lower.
+
+### `systemPreferences.hasFullMediaAccess()` _macOS_
+
+Returns `Boolean` - `true` if the app has obtained system privacy consent to access both the camera and the microphone, and `false` otherwise.
+
+This user consent was not required until macOS 10.14 Mojave, so this method will always return `true` if your system is running 10.13 High Sierra or lower.
+
+### `systemPreferences.askForMediaAccess(askAgain, mediaType)` _macOS_
```

Comment:
Let's pass `mediaType` as a first argument?
It sounds slightly better in that order
```js
askForMediaAccess('camera', /* askAgain */ true)

askForMediaAccess(/* askAgain */ true, 'camera')
```